content/building/build-esp32.md

```diff
@@ -217,6 +217,13 @@ The above may have some errors if:
 
     it will ask you for the COM port where it's connected
 
+1. An other alternative using [nanoff](../getting-started-guides/getting-started-managed.md#uploading-the-firmware-to-the-board-using-nanofirmwareflasher) tool:
```

Comment:
Grammatically,  it might be better to change this sentence to read: "An alternative to using the [nanoff] tool"